src/main.py

```diff
@@ -15,6 +16,12 @@
 SLACK_WEBHOOK = os.environ.get('SLACK_WEBHOOK')
 assert SLACK_WEBHOOK != None, "Failed to get required environment variable SLACK_WEBHOOK"
 
+# Mailer properties
+RECIPIENTS_DICT = json.loads(os.environ.get('RECIPIENTS_DICT', ''))
```

Comment:
Are these required env vars? If so I might suggest adding an `assert VAR != None` to catch it.

Reply:
Not required - modified the code slightly to make sure it can work without these vars.